checks/signed_releases_test.go

```diff
@@ -94,6 +94,25 @@ func TestSignedRelease(t *testing.T) {
 				Score: 10,
 			},
 		},
+		{
+			name: "Releases with assests with intoto SLSA provenance",
+			releases: []clients.Release{
+				{
+					TagName:         "v1.0.0",
+					URL:             "http://foo.com/v1.0.0",
+					TargetCommitish: "master",
+					Assets: []clients.ReleaseAsset{
+						{
+							Name: "foo.asc",
```

Comment:
Shouldn't the name be `foo.intoto.jsonl` as well?

Reply:
well, ideally yes. But it's possible to have one attestation attesting to multiple artifacts. I know it's not super accurate. The proper way is to verify it, we can try that in a follow-up PR to beef it up.

Reply:
I think this test case isn't testing the right thing, from [this line](https://github.com/ossf/scorecard/pull/2131/files#diff-ec7162da628e8587757286bab0848563d1be93f39b0e33facc307792d27ffe94L53) it looks like the `asset.Name` is what is being checked for the suffix, not the URL. Given this I believe this test case is passing only because the `Name` is `*.asc`, not due to `.intoto.jsonl`?

Reply:
+1, you're correct. Updated. Thanks for catching.